community/learn/graphql-tutorials/tutorials/typescript-react-apollo/tutorial-site/content/queries.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Queries"
+metaTitle: "Integrate Queries | GraphQL React Apollo Typescript Tutorial"
+metaDescription: "Integrate GraphQL Queries in React app to fetch personal todo data and handle loading or error state."
+---
+
+Let's integrate a GraphQL query to show the user their personal task list.
+We will learn the following concepts:
+
+- Fetching data with queries using `useQuery` hook
```

Comment:
Separate the bullets into:
- The `Fetch Todos` Query
- Using the query to load data using `useQuery` hooks
- Handling loading/error states
- Mapping types

Reply:
Similar for rest of the sections.